src/config/options.rs

```diff
@@ -216,6 +216,97 @@ pub enum Verbosity {
     Quiet,
 }
 
+/// Which comments to wrap
+#[derive(Copy, Clone, Debug, Eq, PartialEq, Serialize)]
+pub enum WrapComments {
+    /// Don't wrap comments
+    Off,
+    /// Wrap all kinds of comments
+    All,
+    /// Only wrap doc comments
+    Doc,
+    /// Only wrap normal comments
+    Normal,
+}
+
+impl WrapComments {
+    pub(crate) fn is_normal(self) -> bool {
+        matches!(self, WrapComments::All | WrapComments::Normal)
+    }
+
+    pub(crate) fn is_doc(self) -> bool {
+        matches!(self, WrapComments::All | WrapComments::Doc)
+    }
+}
+
+impl fmt::Display for WrapComments {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        match self {
+            WrapComments::Off => f.write_str("Off"),
+            WrapComments::All => f.write_str("All"),
+            WrapComments::Doc => f.write_str("Doc"),
+            WrapComments::Normal => f.write_str("Normal"),
+        }
+    }
+}
+
+impl super::ConfigType for WrapComments {
+    fn doc_hint() -> String {
+        "[Off|All|Doc|Normal]".to_owned()
+    }
+
+    fn stable_variant(&self) -> bool {
+        true
+    }
+}
```

Comment:
Can we move the impl for `ConfigType` to `src/config/config_type.rs` to keep this consistent with other options that manually implement the trait.

Also, should the doc hint include `true` and `false`? Maybe something like this:
```
[ Off (false) | All (true) | Doc | Normal ]
```

Reply:
We could probably drop the `fn stable_variant(&self);` implementation since this is already the default for `ConfigType`.